### PR TITLE
Add recipe for Neobolt

### PR DESCRIPTION
**Pull request by thebestnom**
neobolt is a dependency of neo4j-python-driver from version 1.7 and up, and it looks like it make neo4j-python-driver a noarch package

### CODE REVIEW

**Comment by thebestnom** [comment]
Seems good to go for review @conda-forge/help-python

File: recipes/neobolt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - neobolt
+    - neobolt.direct
+    - neobolt.exceptions
+    - neobolt.routing
+
+about:
+  home: http://neobolt.readthedocs.io
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (chrisburr) [inline recipes/neobolt/meta.yaml]: There is no need to explicitly state the recipe directory, conda will automatically look there if it isn't found in the build directory:
```suggestion
  license_file: LICENSE
```

**Comment by thebestnom** [comment]
Ok, fixed from your suggestion!

**Comment by thebestnom** [comment]
Just another thing, I see that no one maintain neo4j-python-driver, I would like to maintain it

**Comment by chrisburr** [comment]
Once `neobolt-feedstock` has been created you'll get an invite to join the conda-forge GitHub organisation. You can then create a PR to `neo4j-python-driver-feedstock` to add yourself to the list of maintainers and ping* @ conda-forge/core to ask someone to merge it. You could also ask on the [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io).

* Without the space after the @ symbol. Unfortunately GitHub limitations mean you can't ping teams until you're already a member of the organisation, but you will be very soon now I've merged this.

**Pull request merged**